tests/publiccloud/run_ltp.pm

```diff
@@ -121,15 +125,67 @@ sub cleanup {
     type_string('', terminate_with => 'ETX');
 
     upload_logs('ltp_log.raw', failok => 1);
-    parse_extra_log(LTP => "$root_dir/ltp_log.json") if (script_run("test -f $root_dir/ltp_log.json") == 0);
-
+    if (script_run("test -f $root_dir/ltp_log.json") == 0) {
+        my $known_issues = download_whitelist();
```

Comment:
I decided to keep it here  , purpose is that if there is common problem with getting known issues we will not hit it as many times as we have tests

Reply:
Note for other reviewers: `download_whitelist` returns `undef` if the LTP_KNOWN_ISSUES setting is not set.

Reply:
yeah and this is exactly what I am expecting here . so in next line if `$known_issue` is undef we will fully skip everything related to `LTP_KNOWN_ISSUES`